scripts/test-math.ts

```diff
@@ -0,0 +1,53 @@
+import { Network } from "@invariant-labs/sdk-eclipse";
+import { createSnapshotForNetwork } from "../src/snap-points";
+import { IPointsJson } from "../src/types";
+import * as fs from "fs";
+import path from "path";
+import { BN } from "@coral-xyz/anchor";
+import { POINTS_PER_SECOND } from "../src/math";
+
+const testMath = async () => {
```

Comment:
the file name does not indicate what it is testing. Rename it to `validate-points-distribution.ts` please